common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2997,6 +2997,9 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVE_TXN_READONLY_ENABLED("hive.txn.readonly.enabled", false,
       "Enables read-only transaction classification and related optimizations"),
 
+    HIVE_TXN_LOCKLESS_READS_ENABLED("hive.txn.lockless.reads.enabled", false,
```

Comment:
I feel like it would be better to have a different config for non_blocking drop table and partition, they are work quit differently, I think there might be cases when you want to turn off the one, but not the other.

Reply:
makes sense. i'll create a separate config for async drop, however HIVE_TXN_LOCKLESS_READS_ENABLED could be still leveraged to enable whole lockless read feature.